test/e2e/common/configmap_volume.go

```diff
@@ -184,7 +184,12 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 		Eventually(pollLogs, podLogTimeout, framework.Poll).Should(ContainSubstring("value-2"))
 	})
 
-	It("binary data should be reflected in volume [NodeConformance]", func() {
+	/*
+		Release: v1.12
+		Testname: configmap-supports-binary-data
```

Comment:
This should follow a different format, eg: `ConfigMap Volume, binary data`

Reply:
Done